.github/workflows/ci.yml

```
@@ -0,0 +1,33 @@
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php: [7.4, 8.0, 8.1, 8.2, 8.3]
+
+    name: PHP ${{ matrix.php }} Test
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)